api/spec/controllers/spree/api/line_items_controller_spec.rb

```diff
@@ -123,23 +123,23 @@ module Spree
 
         before { order.create_proposed_shipments }
 
-        it "clear out shipments on create" do
-          expect(order.reload.shipments).not_to be_empty
-          api_post :create, :line_item => { :variant_id => product.master.to_param, :quantity => 1 }
-          expect(order.reload.shipments).to be_empty
-        end
-
-        it "clear out shipments on update" do
-          expect(order.reload.shipments).not_to be_empty
-          api_put :update, :id => line_item.id, :line_item => { :quantity => 1000 }
-          expect(order.reload.shipments).to be_empty
-        end
-
-        it "clear out shipments on delete" do
-          expect(order.reload.shipments).not_to be_empty
-          api_delete :destroy, :id => line_item.id
-          expect(order.reload.shipments).to be_empty
-        end
+      #   it "clear out shipments on create" do
+      #     expect(order.reload.shipments).not_to be_empty
+      #     api_post :create, :line_item => { :variant_id => product.master.to_param, :quantity => 1 }
+      #     expect(order.reload.shipments).to be_empty
+      #   end
+
+      #   it "clear out shipments on update" do
+      #     expect(order.reload.shipments).not_to be_empty
+      #     api_put :update, :id => line_item.id, :line_item => { :quantity => 1000 }
+      #     expect(order.reload.shipments).to be_empty
+      #   end
+
+      #   it "clear out shipments on delete" do
+      #     expect(order.reload.shipments).not_to be_empty
+      #     api_delete :destroy, :id => line_item.id
+      #     expect(order.reload.shipments).to be_empty
+      #   end
```

Comment:
I'm curious if anyone can shed light on these tests. I suspect they are basically tests for the existing behavior of deleting the shipments. If we recalculate the shipments, then we should be OK in the overall state of the order, right?